lightning/glue/glue.go

```diff
@@ -126,7 +162,11 @@ func (e ExternalTiDBGlue) GetTables(context.Context, string) ([]*model.TableInfo
 }
 
 func (e ExternalTiDBGlue) GetSession() (checkpoints.Session, error) {
-	return nil, errors.New("ExternalTiDBGlue doesn't have a valid GetSession function")
+	conn, err := e.db.Conn(context.Background())
```

Comment:
I forget to pass a context to this interface. Could you help change this method of glue interface to `GetSession(context.Context)`? (I'll change TiDB implementation of glue in another PR)

Reply:
Todo

Reply:
Done